### PR TITLE
Nested collection limit

### PR DESCRIPTION
**Pull request by Athira001**
Set nested collection limit for individual collection
quintype/ace-planning#18

Dependencies # (dependency-issue-reference)

Documentation - https://github.com/quintype/malibu/compare/nested-collection-doc?expand=1

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test Case A
- [ ] Test Case B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/isomorphic/components/get-collection-template.js [hunk 1]
```diff
@@ -3,3 +3,21 @@ import templates from "./collection-templates";
...
+}
+
+const collectionLimits = Object.entries(templates).reduce(
+  (acc, [key, value]) => Object.assign(acc, { [key]: value.nestedCollectionLimit }),
+  {}
+);
+
+export function getNestedCollectionLimit() {
+  return collectionLimits;
+}
```
Comment (nandap4790) [inline app/isomorphic/components/get-collection-template.js:23]: You might want to update the docs extensively for others to follow the practice

**Review by nandap4790 — APPROVED** [review]

File: app/isomorphic/components/get-collection-template.js (hunk 1 above)
Comment (Harshith46) [reply app/isomorphic/components/get-collection-template.js:23]: yes

File: package.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
     "jest": "^26.0.1",
     "lint-staged": "^10.0.0",
     "mocha": "^8.0.1",
-    "nodemon": "^2.0.2",
+    "nodemon": "^1.8.1",
```
Comment (ags1773) [inline package.json:45]: Why?

Comment (nandap4790) [reply package.json:45]: @ags1773 there is an issue with nodemon running in a loop

**Review by ags1773 — APPROVED** [review]

**Pull request merged**